### DISCUSSION TITLE
Problem with Rainbow RN app when creating/importing wallet

### DISCUSSION THREAD

**Discussion by jvazgot** [rainbow-me/rainbowkit#560] [Feedback]
I've been trying to fork the rainbow app and use it on IOS. But I've been having a problem with creating/importing wallet. It throws this error. Is there something specific I need on the .env for it to work? I can't seem to find the issue.

![image](https://user-images.githubusercontent.com/97035901/176774630-86f13462-01c6-467a-a381-9fb18c6e8f88.png)

**Comment by nebo1234** [comment]
Hello Kindly send a direct message to get assistance
